src/test/vstart_wrapper.sh

```diff
@@ -16,6 +16,7 @@
 # GNU Library Public License for more details.
 #
 
+CEPH_ROOT=${CEPH_ROOT:-..}
```

Comment:
@tchaikov @alimaredia
I'm going to redo this over the weekend.
Probably a combi of multiple direcoty selection and some readline